tools/VersionCop/Program.cs

```diff
@@ -419,8 +419,9 @@ class Program
 
                             Console.WriteLine();
                             Console.WriteLine("*****************************************************************");
-                            Console.WriteLine($"Found in nuspec, but declared dependency version doesn't match!");
-                            Console.WriteLine($"expecting '{packageVersion}' but found '{idFoundInNuspec}'");
+                            Console.WriteLine($"DEPENDENCY VERSION MISMATCH!");
```

Comment:
Comment on this (and line 486 below): this style (ALL CAPS) is not being used anywhere as it doesn't help reading. 
Better use the general pattern followed in our tools: "ERROR: this-is-the-explanation-on-the-error"